apps/meteor/ee/server/lib/audit/endpoints.ts

```diff
@@ -0,0 +1,76 @@
+import type { IUser } from '@rocket.chat/core-typings';
+import { Rooms } from '@rocket.chat/models';
+import type { PaginatedRequest, PaginatedResult } from '@rocket.chat/rest-typings';
+import Ajv from 'ajv';
+
+import { API } from '../../../../app/api/server/api';
+import { getPaginationItems } from '../../../../app/api/server/helpers/getPaginationItems';
+import { findUsersOfRoom } from '../../../../server/lib/findUsersOfRoom';
+
+const ajv = new Ajv({
+	coerceTypes: true,
+});
+
+type AuditRoomMembersParams = PaginatedRequest<{
+	roomId: string;
+	filter: string;
+}>;
+
+const auditRoomMembersSchema = {
+	type: 'object',
+	properties: {
+		roomId: { type: 'string' },
```

Comment:
```suggestion
		roomId: { type: 'string', minLength: 1 },
```
Using `minLength` avoids finding a random room in the DB, which could be dangerous

Reply:
Good one.

Reply:
Note: in this case, this won't find a "random room". The param `roomId` will end up as `''` as it's required. And then `Rooms.findOneById` will issue `{ _id: '' }` which won't match any room and return `null`.

I'm removing the non-obvs behavior but felt good to clarify this part 👀 

If we did something like:
```
findOne({ ...(_id && { _id })
then when _id is falsy it will issue a `findOne({})` which will return the first room in the db. Since here we're passing the param to the call, it won't happen.